windows/clangw/llvm-arw.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+
+int main(int argc, char** argv)
+{
+    // Loop through each argument and transform \ to /.
+    std::string cmd = "llvm-ar";
+    for (int i = 1; i < argc; ++i)
+    {
+        std::string arg = argv[i];
+        for (size_t c = 0; c < arg.size(); ++c)
```

Comment:
If we're using `std::string`, then `std::replace(arg.begin(), arg.end(), '\\', '/')` is probably the more idomatic way to do this.

Reply:
Good point. Fixed.